opengrok-indexer/src/main/java/org/opengrok/indexer/web/Scripts.java

```diff
@@ -86,27 +89,27 @@ public FileScript(String script, int priority) {
 
         @Override
         public String toHtml() {
-            StringBuilder builder = new StringBuilder();
-            builder.append("\t<script type=\"text/javascript\" src=\"");
-            builder.append(this.getScriptData());
-            builder.append("\" data-priority=\"");
-            builder.append(this.getPriority());
-            builder.append("\"></script>\n");
-            return builder.toString();
+            return "\t<script type=\"text/javascript\" src=\"" +
+                    this.getScriptData() +
+                    "\" data-priority=\"" +
+                    this.getPriority() +
+                    "\"></script>\n";
```

Comment:
I didn't know + was better than a builder

Reply:
better = faster ?

Reply:
String concatenation uses StringBuilder in the backend so there is no performance gain but it's shorter this way.

Reply:
In this case the builder approach was slightly more readable for me however that's probably just a matter of taste.

Reply:
This was suggested by IDEA and I thought removing the clutter is better but I can revert it back :)

Reply:
Ok